snap/local/create-cluster.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+NAME=$1
+
+PG_MAJOR_VER=$(echo "$SNAP_VERSION" | cut -d "." -f 1)
+
+# TODO figure out local-auth
```

Comment:
del?

Consider to allow localhost.

Consider to use `set -e` in all shell scripts to abort execution on the exception.

Reply:
Unfortunately, localhost uses the `auth-host` parameter. `--auth-local=trust` means that everyone can connect locally using the socket by default.